src/iop/colorchecker.c

```diff
@@ -1327,31 +1327,31 @@ void gui_init(struct dt_iop_module_t *self)
   dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, g->combobox_patch);
 
   g->scale_L = dt_bauhaus_slider_new_with_range(self, -100.0, 200.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_L, _("lightness offset"));
+  gtk_widget_set_tooltip_text(g->scale_L, _("scale lightness in Lab space\nlower values darken target color while higher brighten it"));
   dt_bauhaus_widget_set_label(g->scale_L, NULL, N_("lightness"));
 
   g->scale_a = dt_bauhaus_slider_new_with_range(self, -256.0, 256.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_a, _("chroma offset green/red"));
-  dt_bauhaus_widget_set_label(g->scale_a, NULL, N_("green/red"));
+  gtk_widget_set_tooltip_text(g->scale_a, _("scale target color in Lab a dimmension\nlower values shift target color towards greens while higher shift towards magentas"));
+  dt_bauhaus_widget_set_label(g->scale_a, NULL, N_("green-magenta offset"));
   dt_bauhaus_slider_set_stop(g->scale_a, 0.0, 0.0, 1.0, 0.2);
   dt_bauhaus_slider_set_stop(g->scale_a, 0.5, 1.0, 1.0, 1.0);
   dt_bauhaus_slider_set_stop(g->scale_a, 1.0, 1.0, 0.0, 0.2);
 
   g->scale_b = dt_bauhaus_slider_new_with_range(self, -256.0, 256.0, 1.0, 0.0f, 2);
-  gtk_widget_set_tooltip_text(g->scale_b, _("chroma offset blue/yellow"));
-  dt_bauhaus_widget_set_label(g->scale_b, NULL, N_("blue/yellow"));
+  gtk_widget_set_tooltip_text(g->scale_b, _("scale target color in Lab b dimmension\nlower values shift target color towards blues while higher shift towards yellows"));
```

Comment:
Lab b is read ok, but will need to be made consistent with whatever is done for Lab a. above.